mycli/main.py

```diff
@@ -170,6 +173,12 @@ def execute_from_file(self, arg, **_):
 
         return self.sqlexecute.run(query)
 
+    def execute_system_command(self, arg, **_):
+          if not arg:
+              message = 'Missing required argument: command.'
+              yield(None, None, None, message)
+          yield(None, None, None, subprocess.call(arg, shell=True))
+
```

Comment:
This function shouldn't live in main.py. But instead should be transferred to https://github.com/dbcli/mycli/blob/master/mycli/packages/special/iocommands.py

I believe it should also use the @special_command() decorator instead of using the register_special_command() call.

The `register_special_command()` call is a way to write special commands that rely on things that are beyond the args and cursor, for example all the special command handlers in main.py uses the `self` variable in one way or another. Which is a good indication that we need those handlers to be part of this class. But the `execute_system_command` doesn't use the self variable that it takes in as an argument. Which is a good indication that it doesn't need to be part of this class.